src/analysis/openstreetmap/qgsosmdatabase.h

```diff
@@ -82,7 +82,6 @@ class ANALYSIS_EXPORT QgsOSMDatabase
 
     QgsOSMTags tags( bool way, QgsOSMId id ) const;
 
-    //! \note available in Python bindings
```

Comment:
No, it **is** available. It's the default behaviour. I only removed the comment saying it's available.